apps/server/src/shared/repo/user/user.repo.ts

```diff
@@ -146,6 +146,13 @@ export class UserRepo extends BaseRepo<User> {
 		return [users, count];
 	}
 
+	async update(user: User): Promise<User> {

```

Comment:
This method is not needed. This should be done in the use case. Just fetch the entity, update its properties and then persist it using the repository like (in the uc):
```typescript
const targetUser = await this.userRepo.findById(targetUserId, true);
targetUser.forcePasswordChange = true;
// no need for a return value because the targetUser is in the Unit of Work
await this.userRepo.save(targetUser);
```